### DISCUSSION TITLE
New Render2Textures shows wrong labels on hover inputs

### DISCUSSION THREAD

**Discussion by bbbbbgdn** [cables-gl/cables_docs#548] [! Welcome - readme first!]
(visual bug besides works wrong labels works fine)

The new Render2Textures (which is with multiple textures output) shows wrong labels on hovering inputs:
(1 ok) use viewport size
(2) pixel format index
(3) filter index
(4) wrap index
...
<img width="430" alt="Screenshot 2023-07-09 at 11 59 21" src="https://github.com/cables-gl/cables_docs/assets/12091461/f73544ac-54e7-4a43-b046-6fa6ee3382c9">

**Comment by constructor** [comment]
'Height' and 'Width' do not have inputs, not sure if this is intentional or not.
When wiring up an 'Integer' to input 'Pixel Format Index', the wire is graphically displayed as being connected to the input 2 places to the right, and the input connector height is increased slightly. This seems to happen with other inputs also.

![Render2Textures](https://github.com/cables-gl/cables_docs/assets/176053/68921c10-98cf-4305-9e37-e67b6754d851)

**Comment by pandrr** [comment]
thanks. fixed on dev.cables.gl
